.github/workflows/pull.yml

```diff
@@ -362,14 +362,14 @@ jobs:
       test-matrix: ${{ needs.linux-focal-py3_9-clang9-xla-build.outputs.test-matrix }}
     secrets: inherit
 
-  win-vs2019-cpu-py3-build:
+  win-vs2022-cpu-py3-build:
```

Comment:
Alas, this is a no-op, as VC version is defined by AMI

Reply:
Yeah, we are going to clean up the AMI next